### ISSUE TITLE
Invalid endpoint

### ISSUE THREAD

**Issue by Sibghata** [slametps/MMM-RandomQuranAyah#8]
Salam,
The endpoint is no longer valid.

**Comment by slametps** [comment]
which endpoint?

for instance: `http://api.alquran.cloud/ayah/5/editions/ar` is still valid and return a valid result.

**Comment by Sibghata** [comment]
Okay let me try again, I'll get back to you :-)

**Comment by Sibghata** [comment]
Yeah both endpoints (to the ayah and to the translation works fine).
But Inside MM2 it is just loading.. and nothing happens.

**Comment by Sibghata** [comment]
So, alhamdulillah, I got working!
It was due to the fact that the example used ' as text identifier (for the config.js file), while it was expecting ".
Problem solved.
Jzk allah for the great module! May Allah reward for the work you put into it, and for spreading the light of the quran!

**Comment by slametps** [comment]
Alhamdulillah. Waiyyaka.
